### PR TITLE
update rollbar

### PR DESCRIPTION
**Pull request by jdoconnor**
@11mdlow @lumoslabs/platform

### CODE REVIEW

**Comment by 11mdlow** [comment]
what's the deal with hound?

File: Gemfile [hunk 1]
```diff
@@ -7,11 +7,3 @@ gem "jquery-rails"
 
 # Airbrake must be required before initialization because its middleware is added via an initializer
 gem 'airbrake'
-gem 'rollbar'
-
-group :development, :test do
-  gem 'pry'
-  gem 'pry-nav'
-  gem 'pry-rescue'
-  gem 'pry-stack_explorer'
```
Comment (11mdlow) [inline Gemfile]: why did all the pry stuff get taken out? I feel like we have those in our other `:development, :test` groups

Comment (jdoconnor) [inline Gemfile]: still in the gemspec
https://github.com/lumoslabs/smoke_detector/blob/master/smoke_detector.gemspec#L29-L32

Comment (11mdlow) [inline Gemfile]: ah, got it

File: spec/support/shared_examples_for_providers.rb [hunk 2]
```diff
@@ -32,7 +32,10 @@
 shared_examples_for 'Rollbar integrated error handler' do
   context 'uncaught in a controller' do
     it 'reports the error to Rollbar' do
-      Rollbar.should_receive(:log).with('error', instance_of(RuntimeError)).and_return({})
+      Rollbar.should_receive(:log) do |level, args|
+        expect(level).to eq('error')
```
Comment (jcwilk) [inline spec/support/shared_examples_for_providers.rb]: double quotes suck

FTFY

Comment (seunghyo) [inline spec/support/shared_examples_for_providers.rb]: sweet jesus it's john

Comment (jcwilk) [inline spec/support/shared_examples_for_providers.rb]: always lurking for the perfect bot belittlement opportunity

**Comment by 11mdlow** [comment]
:+1:

**Pull request merged**